pyiceberg/catalog/rest/__init__.py

```diff
@@ -289,6 +254,26 @@ def _create_session(self) -> Session:
 
         return session
 
+    def _create_legacy_oauth2_auth_manager(self, session: Session) -> AuthManager:
+        """Create the LegacyOAuth2AuthManager by fetching required properties.
+
+        This will be deprecated in PyIceberg 1.0
```

Comment:
Deprecated or removed in 1.0? It is a private function, so we should be able to remove them any time.

Reply:
removed sounds right :)